Common/AnatupleProducer.py

```diff
@@ -103,8 +103,11 @@ def addAllVariables(df, syst_name, isData, trigger_class):
     return df
 
 
-def createAnatuple(inFile, outFile, period, sample, X_mass, snapshotOptions,range, isData, evtIds, isHH, triggerFile,
+def createAnatuple(inFile, outFile, config, sample_name, mass, snapshotOptions,range, isData, evtIds, isHH, triggerFile,
```

Comment:
mass, isData, isHH, triggerFile all can be taken from config

Reply:
ok